test/testcafe/framework/page-objects/ChallengeOktaVerifyPushPageObject.js

```diff
@@ -24,10 +26,16 @@ export default class ChallengeOktaVerifyPushPageObject extends ChallengeFactorPa
   }
 
   getResendPushButton() {
+    if (userVariables.v3) {
+      return this.form.getButton(RESEND_PUSH_NOTIFICATION_TEXT)
+    }
     return this.form.getElement('.button-primary');
   }
 
   clickResendPushButton() {
+    if (userVariables.v3) {
+      return this.form.clickSaveButton(RESEND_PUSH_NOTIFICATION_TEXT)
```

Comment:
Can we use `this.t.click(this.getResendPushButton())`? Should be able to replace both parts of this in one.